core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -106,56 +107,71 @@ class BpmBillingProjectLifecycle(
         )
       })
 
+    def addMembersToBillingProfile(profileModel: ProfileModel): Future[Set[Unit]] = {
+      val members = createProjectRequest.members.getOrElse(Set.empty)
+      Future.traverse(members) { member =>
+        Future(blocking {
+          billingProfileManagerDAO.addProfilePolicyMember(profileModel.getId,
```

Comment:
Can you add test coverage for this?

Reply:
@MatthewBemis I think this is the last remaining item.

Reply:
Done in 06f85f5a7